### PR TITLE
Add Presets Option

### PR DESCRIPTION
**Pull request by quickliketurtle**
Add option to pass either a single or list of presets separated by a comma. 
`lambo superApplication --presets "base"` or `lambo superApplication --presets "base,API"`

This will look for a file called `~/.lambo/presets/PRESET.sh` and run it if it exists, or display a warning if it does not exist. 

In this gif you can see the output with the preset existing 'base', in this case just an echo statement, and a preset not existing 'API'.
lambopresets

### CODE REVIEW

**Comment by mattstauffer** [comment]
@quickliketurtle Great work! A few thoughts:

- I wonder if we rename it `recipes` that would fix the "no short code" problem; `-r`?
- I think I'd like it to error out immediately if the preset/recipe doesn't exist. That way you avoid waiting for the whole damn thing to run only to figure out you typo'ed

Other than that, this is awesome!

**Comment by quickliketurtle** [comment]
@mattstauffer I like `recipes`, they sound more delicious than `presets`. 

My thought on displaying an error at the end rather than aborting the script if a file is missing was that nothing else will 'break', and you could run the recipe manually after you fix the issue... 
But aborting if the recipe does not exist makes sense, wasn't thinking about the inevitable type. 

I'll make both those changes.  

Thanks.

**Comment by quickliketurtle** [comment]
@mattstauffer Both changes have been made. 
One issue that came up is using abort() is displaying the error text twice. I think i have the fix but wanted to test more and address that separately.

**Comment by quickliketurtle** [comment]
Here is the new output.
lambopresets

File: lambo [hunk 1]
```diff
@@ -267,6 +272,16 @@ if ! hash git 2>/dev/null; then
   quit
 fi
 
+### Check if preset files exist
+IFS=',' read -ra RECIPES <<< "$RECIPES"
+
+for RECIPE in "${RECIPES[@]}"; do
+    if [[ ! -f ~/.lambo/recipes/$RECIPE.sh ]]; then
+        echo "${red}Recipe $RECIPE not found. ${reset}"
+        abort;
```
Comment (milose) [inline lambo]: You should use `quit` I guess, as it's used everywhere else.

**Comment by milose** [comment]
On a funny side, since lambo is a car, I'd rather see this functionality ending up being called `parts` or `kits` 😆

**Comment by mattstauffer** [comment]
The idea is great. I think I want to move all of our future development into the PHP side, though (testable!!) so I think I'm going to hold off on all merges until we move forward there.

It's my fault for waiting so long to merge this--it should've been merged in long before the PHP branch split out.

**Pull request closed without merging**